utilities/calculateFinalPot.test.ts

```diff
@@ -0,0 +1,190 @@
+    heroIndex: 0,
+    playerCards: [
+      {
+        left: {
+          rank: Rank.seven,
+          suit: Suit.heart
+        },
+        right: {
+          rank: Rank.eight,
+          suit: Suit.diamond
+        }
+      },
+      null
+    ],
+    smallBlindSize: 0.05,
+    antiSize: 0,
+    preflop: {
+      actions: [
+        {
+          playerIndex: 0,
+          betSize: 0
+        },
+        {
+          playerIndex: 1,
+          betSize: 0.025
+        }
+      ]
+    }
+  };
+
+  const gameSituationData1 = {
+    ...baseGameSituationData,
+    preflop: {
+      actions: [
+        {
+          playerIndex: 0,
+          betSize: 0.05
+        },
+        {
+          playerIndex: 1,
+          betSize: 0.05
+        },
+        {
+          playerIndex: 0,
+          betSize: 0
+        },
+        {
+          playerIndex: 1,
+          betSize: 0
+        }
+      ]
+    },
+    flop: {
+      communityCards: {
+        left: {
+          suit: Suit.diamond,
+          rank: Rank.five
+        },
+        center: {
+          suit: Suit.diamond,
+          rank: Rank.six
+        },
+        right: {
+          suit: Suit.diamond,
+          rank: Rank.seven
+        }
+      },
+      actions: [
+        {
+          playerIndex: 0,
+          betSize: 1.0
+        },
+        {
+          playerIndex: 1,
+          betSize: 0
+        }
+      ]
+    }
+  };
+
+  const gameSituationData2 = {
+    ...gameSituationData1,
+
+    flop: {
+      ...gameSituationData1.flop,
+      actions: [
+        {
+          playerIndex: 0,
+          betSize: 1.0
+        },
+        {
+          playerIndex: 1,
+          betSize: 1.0
+        },
+        {
+          playerIndex: 0,
+          betSize: 0
+        },
+        {
+          playerIndex: 1,
+          betSize: 0
+        }
+      ]
+    },
+    turn: {
+      communityCard: {
+        suit: Suit.diamond,
+        rank: Rank.five
+      },
+      actions: [
+        {
+          playerIndex: 0,
+          betSize: 1.0
+        },
+        {
+          playerIndex: 1,
+          betSize: 0
+        }
+      ]
+    }
+  };
+
+  const gameSituationData3 = {
+    ...gameSituationData2,
+
+    turn: {
+      ...gameSituationData2.turn,
+      actions: [
+        {
+          playerIndex: 0,
+          betSize: 1.0
+        },
+        {
+          playerIndex: 1,
+          betSize: 1.0
+        },
+        {
+          playerIndex: 0,
+          betSize: 0
+        },
+        {
+          playerIndex: 1,
+          betSize: 0
+        }
+      ]
+    },
+    river: {
+      communityCard: {
+        suit: Suit.diamond,
+        rank: Rank.seven
+      },
+
+      actions: [
+        {
+          playerIndex: 0,
+          betSize: 1.0
+        },
+        {
+          playerIndex: 1,
+          betSize: 0
+        }
+      ]
+    }
+  };
+
+  it("returns final pot when ended street is preflop", () => {
+    expect(calculateFinalPot(baseGameSituationData)).toBe(0.025);
```

Comment:
The same thing happened with https://github.com/agnosticful/chipstackoverflow-web/pull/128#discussion_r396835168

Reply:
@axross Thank you for your comment. However, baseGameSituationData is used twice actually. ( It used at line 41 and 176). Only gameSituationData3 was used once and I knew it but I did it for readability.  What do you think?

Reply:
I’m sorry I overlooked it. Then please only change the identifier to be proper one.

Reply:
Thanks